src/styles/blocks/content.scss

```diff
@@ -0,0 +1,16 @@
+.content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  flex-wrap: wrap;
+
+  @media (min-width: 1200px) {
+    padding: 50px 128px;
+  }
+
+  margin: 0 auto;
```

Comment:
about this and **CARD.scss line 7**
look at this checklist, there are photo how should it look like when it will be less than 8 cards. 

CHECKLIST
13. [LAYOUT] - don't hardcode max-width, your catalog container should work fine for any number of cards